plugins/event_source/aws_sqs_queue.py

```diff
@@ -18,16 +28,35 @@
 from aiobotocore.session import get_session
 
 
+def connection_args(args: Dict[str, Any]) -> Dict[str, Any]:
+    selected_args = {}
+
+    # Best Practice: get credentials from ~/.aws/credentials or the environment
+    # However the following method are still possible
+    if "access_key" in args:
+        selected_args["aws_access_key_id"] = args["access_key"]
+    if "secret_key" in args:
+        selected_args["aws_secret_access_key"] = args["secret_key"]
+    if "session_token" in args:
+        selected_args["aws_session_token"] = args["session_token"]
+
+    if "endpoint_url" in args:
+        selected_args["endpoint_url"] = args["endpoint_url"]
+    if "region" in args:
+        selected_args["region_name"] = args["region"]
+    return selected_args
+
+
 async def main(queue: asyncio.Queue, args: Dict[str, Any]):
     logger = logging.getLogger()
 
-    region = args.get("region")
-    queue_name = args.get("queue")
-    wait_seconds = int(args.get("wait_seconds", 2))
+    if "name" not in args:
+        raise RuntimeError("Missing queue name")
```

Comment:
Here is more appropriate the `ValueError` exception: https://docs.python.org/3/library/exceptions.html#ValueError